deb/scripts/_after-install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games'
```

Comment:
are all of these directories necessary for this script to do its job?  specifically, '/usr/games' ?